crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -240,17 +243,17 @@ mod tests {
         while let Some((spend, rest)) = a.next(all_spends) {
             all_spends = rest;
             // process the spend
-            let [parent_id, puzzle, amount, solution, _spend_level_extra] =
-                extract_n::<5>(&a, spend, ErrorCode::InvalidCondition).expect("extract_n");
-
+            let destructure_tuple!(parent_id, puzzle, amount, solution, _) =
+                <match_tuple!(BytesImpl<32>, Program, u64, Program, NodePtr)>::from_clvm(&a, spend)
```

Comment:
```suggestion
                <match_tuple!(BytesImpl<32>, NodePtr, u64, NodePtr, NodePtr)>::from_clvm(&a, spend)
```

This will prevent the serialization of the `NodePtr` tree into bytes, and be cheaper.